### DISCUSSION TITLE
group claims upper case by default

### DISCUSSION THREAD

**Discussion by brackend** [dexidp/dex#2366] [Q&A] [answered]
I'm using dex connector to control oidc flow between argocd and an idp.  when it returns the groups claims, the groups are all converted to upper case.  I would like to preserve the case but not sure where this is done in the code or if it can be disabled through configuration file.  Anyone any ideas?

**Comment by sagikazarmark** [comment, marked as answer]
It's highly unlikely Dex itself converts the group names to upper case.

**Reply by brackend** [reply]
Yes you are right.  Thank you!  The idp was configured to convert groups to upper case.  I had initially ruled idp out based on misinformation.
